src/App.tsx

```diff
@@ -1,156 +1,75 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { ShowError } from './components/ShowError/ShowError';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList/TodoList';
+import { Header } from './components/Header/Header';
 
-const USER_ID = 0;
+const USER_ID = 6156;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosToRender, setTodosToRender] = useState<Todo[]>([]);
+  const [todoFilter, setTodoFilter] = useState('All');
+  const [error] = useState('');
+
+  const renderClearCompleted
+    = !!(todos.filter(todo => todo.completed).length);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(r => {
+        setTodos(r);
+        setTodosToRender(r);
```

Comment:
never use one-letter names for your variables, I don't understand what you're getting there because of the name`r` . Let's give descriptive names for our variables